mycli/main.py

```diff
@@ -715,15 +705,41 @@ def get_prompt(self, string):
         string = string.replace('\\_', ' ')
         return string
 
-    def run_query(self, query, table_format=None, new_line=True):
+    def run_query(self, query, new_line=True):
         """Runs query"""
         results = self.sqlexecute.run(query)
         for result in results:
             title, cur, headers, status = result
-            output = format_output(title, cur, headers, None, table_format)
+            output = self.format_output(title, cur, headers, None)
             for line in output:
                 click.echo(line, nl=new_line)
 
+    def format_output(self, title, cur, headers, status, expanded=False,
+                      max_width=None):
+        table_format = 'expanded' if expanded else self.table_format
```

Comment:
That's a good point. What do you think of having an optional keyword override for table format as well. For instance:

```python
formatter = OutputFormatter(format_name='simple')
formatter.format_output(data, headers)  # formats as simple
formatter.format_output(data, headers, format_name='expanded')  # formats as expanded
```